concepts/0021-didcomm-message-anatomy/README.md

```diff
@@ -0,0 +1,96 @@
+# 0021: DID Comm Message Anatomy
```

Comment:
I don't think changes from this file should be part of this PR; it looks like we're mixing 2 different concerns.

Reply:
Second this. Can you remove these changes as well as the changes made to the code to a separate PR?

Reply:
How do I remove this polution? Can Daniel or Kyle remove it? Can the poluter remove it?

Reply:
@Oskar-van-Deventer: I will clean up the extra stuff and fix the DCO problem. Please give me until the end of my workday. Then we can merge the PR.